bitnami/phpbb/templates/tls-secrets.yaml

```diff
@@ -5,6 +5,7 @@ kind: Secret
 metadata:
   name: {{ .name }}
   labels: {{- include "common.labels.standard" $ | nindent 4 }}
+    app.kubernetes.io/component: phpbb
     {{- if $.Values.commonLabels }}
     {{- include "common.tplvalues.render" ( dict "value" $.Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
```

Comment:
It'd be great if we could auto-generate a certificate when not provided (see keycloak for instance)